src/docs/build-and-earn/testnet-guides/run-seed-node.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Run a Seed Node
+---
+
+## ﻿Run a Seed Node
+
+Before running a node, make sure that you have followed the guide for [setting up your prerequisites and environment](/docs/build-and-earn/testnet-guides/prerequisites.md).
+
+To run a seed node, you need to complete the following steps:
+
+- Initialize the node
+- Configure and start the node
+
+## Initialize and start the node without being a validator
+
+1. Navigate to the directory *CudosBuilders/docker/seed-node*
+2. Find the file **seed-node.env.example** and create a copy of it
+3. Rename the copied file to **seed-node.client.testnet.public01.env**
+4. Open the file, which you renamed, **seed-node.client.testnet.public01.env** in any editor
+5. Find  the **"MONIKER"** attribute and set a name to it as the following:
+```
+MONIKER = MySeedNodeName
+```
+6. Initiliaze the node by running this command:
+```
+sudo docker-compose --env-file ./seed-node.client.testnet.public01.arg -f ./init-seed-node.yml -p cudos-init-seed-node-client-testnet-public-01 up --build
+```
+7. Start the node by running this command:
+```
+sudo docker-compose --env-file ./seed-node.client.testnet.public01.arg -f ./start-seed-node.yml -p cudos-start-seed-node-client-testnet-public-01 up --build
```

Comment:
i think we can append the ` --detach` switch to this command to let the container run in the background
then additionally we can say user can use:
`sudo docker logs -f cudos-start-seed-node-client-testnet-public-01`
command to see the logs